lib/outputs/outputs.py

```diff
@@ -0,0 +1,78 @@
+from csv import DictWriter
+
+from colorama import Fore, Style
+
+from config.config import csv_header, timestamp
+from lib.outputs.models import Check_Output
+from lib.utils.utils import file_exists, open_file
+
+
+def report(check_findings, output_options, audit_info, organizations_info):
+    check_findings.sort(key=lambda x: x.region)
+    format_timestamp = timestamp.strftime("%Y%m%d%H%M%S")
+    # check output options
+    file_descriptors = {}
+    for output_mode in output_options.output_modes:
+        if output_mode == "csv":
+            filename = (
+                f"prowler-output-{audit_info.audited_account}-{format_timestamp}.csv"
+            )
+            if file_exists(filename):
+                file_descriptor = open_file(
+                    f"prowler-output-{audit_info.audited_account}-{format_timestamp}.csv",
+                    "a",
+                )
+            else:
+                file_descriptor = open_file(
+                    f"prowler-output-{audit_info.audited_account}-{format_timestamp}.csv",
```

Comment:
Include the file suffix name and extension in variables, we are going to need it in other report formats.

Reply:
Included